### PR TITLE
Show unicode below code point #113

### PR DESCRIPTION
**Pull request by carlinmack**
This renders as so
image

I'm not sure where the html template is defined so I can't do the aligning as requested

Also I'm not sure how you would like it to be linked

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -501,6 +501,13 @@ $(document).ready(function () {
     var emoji = isSkintoneVariant ? baseEmoji.emoji : currEmoji.emoji;
     var annotation = isSkintoneVariant ? baseEmoji.annotation : currEmoji.annotation;
     var hexcode = currEmoji.hexcode;
+    var zwj = "";
```
Comment (b-g) [inline src/library.js:504]: - find the var name confusing :)
- can you add the links?

... then I think it is good enough to go online. We can style it later.

Comment (carlinmack) [reply src/library.js:504]: * yes, not sure what the correct terminology should be
* where should we link to?

Comment (b-g) [reply src/library.js:504]: - Maybe `var combination`?
- I would vote to link to us and for ZWJ etc. to a technical resource or https://emojipedia.org/zero-width-joiner/

Comment (carlinmack) [reply src/library.js:504]: 👍👍

**Comment by carlinmack** [comment]
btw I'm not sure how this can be tested but I think it works :)

**Comment by b-g** [comment]
Thanks! Tweaked it further ...

**Pull request merged**